tests/test_modules.py

```diff
@@ -27,19 +27,17 @@
 from parseAndPopulate.modules import SdoModule, VendorModule
 
 
+# TODO: we should probably have unit tests for the individual Model._resolve* methods
```

Comment:
This isn't relevant anymore since the resolves were moved to independent classes. Please remove it to avoid confusion.